modules/update-preparing-ack.adoc

```diff
@@ -19,9 +19,9 @@ Be aware that all responsibility falls on the administrator to ensure that all u
 
 .Procedure
 
-* Run the following command to acknowledge that you have completed the evaluation and your cluster is ready for the Kubernetes API removals in {product-title} 4.12:
+* Run the following command to acknowledge that you have completed the evaluation and your cluster is ready for the Kubernetes API removals in {product-title} 4.13:
 +
 [source,terminal]
 ----
-$ oc -n openshift-config patch cm admin-acks --patch '{"data":{"ack-4.11-kube-1.25-api-removals-in-4.12":"true"}}' --type=merge
+$ oc -n openshift-config patch cm admin-acks --patch '{"data":{"ack-4.12-kube-1.26-api-removals-in-4.13":"true"}}' --type=merge
```

Comment:
@wking @sdodson Do we know for sure yet what this ack will be for 4.13?

Reply:
I am working on it  as part of https://issues.redhat.com/browse/OTA-899 and it would be `ack-4.12-kube-1.26-api-removals-in-4.13`

Reply:
Perfect thanks @LalatenduMohanty!